oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/JwtReactiveAuthenticationManager.java

```diff
@@ -39,6 +42,7 @@
  * @since 5.1
  */
 public final class JwtReactiveAuthenticationManager implements ReactiveAuthenticationManager {
+	private static final Log logger = LogFactory.getLog(JwtReactiveAuthenticationManager.class);
```

Comment:
Looks like this might be unnecessary for now - in fact, I think this class can be removed from the PR